### ISSUE TITLE
breaks with some @typescript-eslint tules

### ISSUE THREAD

**Issue by artola** [Quramy/typescript-eslint-language-service#217]
The language service is not reporting errors, when some of the rules included in `@typescript-eslint` are in use. For example (but not limited) including any of these rules stops reporting other errors as `semi: 2`:

```
  plugins: ['@typescript-eslint'],
  rules: {
   '@typescript-eslint/await-thenable': 2,
   '@typescript-eslint/dot-notation': 2,
   '@typescript-eslint/no-base-to-string': 2,
   '@typescript-eslint/no-confusing-void-expression': 2,
   '@typescript-eslint/no-implied-eval': 2,
  }
```

Version used:

    "@typescript-eslint/eslint-plugin": "^4.21.0",
    "@typescript-eslint/parser": "^4.21.0",
    "@typescript-eslint/typescript-estree": "^4.21.0",
    "typescript-eslint-language-service": "^4.1.3"

@Quramy Do you have any idea/hint about it?

**Comment by Quramy** [comment]
@artola 

I've fixed it and published v4.1.4 , pls try this.

Thanks for your reporting

**Comment by artola** [comment]
@Quramy Awesome 👏  Now all my rules are running properly. Thanks a lot for the package and the support. 🙇
